js/question.js

```diff
@@ -192,7 +192,34 @@ function Question(name, type, required, label, max_length, prefix, slug) {
 
     input.keyup(handler).change(handler).click(handler);
     input.keyup(needs_save).change(needs_save);
-    input.keydown(prevent_prefix_edit).on('cut copy paste', prevent_prefix_edit);
+    input.keydown(prevent_prefix_edit).on('cut', prevent_prefix_edit);
+    input.on('paste', function(e) {
+        var regex;
+        var index;
```

Comment:
I can think about this a bit more/review this for real, but this whole index thing is a bit sketch. You should be able to specify the capture group

Reply:
thinking about this a bit more, you should just be able to use the last group in all of these rather than specifying the index.

Reply:
you could also just use non capturing groups for the stuff you don't need to keep

Reply:
@VjunetXuuftofi this is also true and definitely the way to do this (see my first comment)

Reply:
I became a regex master doing this; https://github.com/VjunetXuuftofi/ElectoralDataProject/blob/master/parser.py